apps/wizarr-frontend/src/modules/admin/components/Users/UserList/UserItem.vue

```diff
@@ -103,6 +103,9 @@ export default defineComponent({
     },
     methods: {
         async getProfilePicture() {
+            if (!this.user.username) {
```

Comment:
blank email needs to be handled on the backend, as it's a issue with plex guest users

Reply:
Handled in #357